extension/js/content_scripts/webmail/gmail_element_replacer.ts

```diff
@@ -130,28 +132,23 @@ export class GmailElementReplacer implements WebmailElementReplacer {
     const visibleReplyBtns = $('td.acX:visible');
     if (visibleReplyBtns.not('.replaced, .inserted').length) { // last reply button in convo gets replaced
       const convoReplyBtnsToReplace = visibleReplyBtns.not('.replaced, .inserted');
-      const hasVisibleReplacements = visibleReplyBtns.filter('.replaced, .inserted').length > 0;
       const convoReplyBtnsToReplaceArr = convoReplyBtnsToReplace.get();
```

Comment:
`convoReplyBtnsToReplaceArr` should be called `convoReplyBtnsArr` (we are not replacing them anymore)